src/components/main-form.js

```diff
@@ -50,6 +50,35 @@ option2.term = 30
 
 // CSS
 
+const Summary = styled(Row)`
+  b {
+    font-weight: 500;
+  }
+
+  p,
+  h5 {
+    font-size: 17px;
+  }
+
+  h5 {
+    margin-bottom: 1rem;
+    margin-left: 1.5rem;
+  }
+
+  .col {
+    margin-left: -2rem;
+  }
+
+  ol {
+    font-size: 15px;
+    margin-right: 2.5rem;
+  }
+
+  li {
+    margin-bottom: 1rem;
+  }
+`
```

Comment:
sug: As discussed, if you wanted to pull these out, styled components would be faster to cmd-f